### PR TITLE
Add gauge example to stress playground

### PR DESCRIPTION
**Pull request by JamesNK**
For testing https://github.com/dotnet/aspire/issues/2884
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/aspire/pull/2910)

### CODE REVIEW

**Comment by davidfowl** [comment]
Unrelated to your change but I had a discussion with an MVP about the fact that the auto scrolling might make the dashboard unusable in situations where there's lots of traffic filling up the circular buffer. 

Have you tried using the dashboard for telemetry when there's continuous load?

**Comment by JamesNK** [comment]
There's been discussion about having a pause button. It would cause the OTLP receiver to ignore incoming data. That would solve the extreme load scenario.

Doubtful for GA.

**Comment by davidfowl** [comment]
Auto scroll only affects the

**Comment by JamesNK** [comment]
If you scroll up then it pauses on the structured logs and traces pages.

**Comment by timheuer** [comment]
> It would cause the OTLP receiver to ignore incoming data

Why would it stop the feed and not just affect the UI.  e.g. thinking of tail logs on things like pihole UI, this is a common pattern...logs still happen, data received, but the UI is just holding scroll behavior

Screenshot 2024-04-06 at 2 54 41 PM

**Comment by JamesNK** [comment]
There are limits on the amount of data received. If you hit pause and data was still being received, then the data you're viewing could disappear if data limits are exceeded.

You can already pause automatic scrolling by scrolling up any amount.

**Comment by JamesNK** [comment]
Review please 🙏

File: playground/Stress/Stress.TelemetryService/GaugeMetrics.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+        var process = Process.GetCurrentProcess();
+
+        var meter = meterFactory.Create("GaugeMetrics");
+        meter.CreateObservableGauge(
+            "ProcessWorkingSetGauge",
+            () =>
+            {
+                var measurements = new List<Measurement<long>>();
+                var processes = Process.GetProcesses().ToList();
```
Comment (davidfowl) [inline playground/Stress/Stress.TelemetryService/GaugeMetrics.cs:23]: Why ToList?

**Review by davidfowl — APPROVED** [review]

**Pull request merged**